src/main/java/com/faforever/client/coop/CoopController.java

```diff
@@ -244,11 +238,7 @@ private CoopMissionBean getSelectedMission() {
   }
 
   private void setSelectedMission(CoopMissionBean mission) {
-    JavaFxUtil.runLater(() -> {
-      descriptionWebView.getEngine().loadContent(mission.getDescription());
-      mapImageView.setImage(mapService.loadPreview(mission.getMapFolderName(), PreviewSize.SMALL));
```

Comment:
Remove the map column as well

Reply:
The column cannot be removed because it is being used elsewhere. I just hid it instead. Also hid rating range column. It is always empty and takes up extra space.

Reply:
![image](https://user-images.githubusercontent.com/38242291/143689075-b83788aa-3011-4aa5-b046-1f3a51dd26df.png)

Reply:
Yeah hiding is good